rest-api/src/test/java/io/quarkus/qe/RepositoryResourceTest.java

```diff
@@ -171,6 +196,20 @@ private void thenResponseErrorCodeIs(int expectedCode) {
     }
 
     private void thenResponseErrorMessageIs(String expectedMsg) {
-        assertTrue(response.as(CatalogError.class).getMsg().equalsIgnoreCase(expectedMsg));
+        var msg = response.as(CatalogError.class).getMsg();
+        var explainErrMsg = String.format("Expected: %s  | found: %s", expectedMsg, msg);
+        assertTrue(response.as(CatalogError.class).getMsg().equalsIgnoreCase(expectedMsg), explainErrMsg);
```

Comment:
I think this is equivalent to: 

```java
assertEquals(expectedMsg, response.as(CatalogError.class).getMsg());
```

The advantage is that JUnit will display the expected/actual message for you in case the test fails.